### ISSUE TITLE
problematic substance use and Addictive non-substance behavior

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10170]
Should both be direct children of [Recurrent maladaptive behavior HP:5200241](https://hpo.jax.org/app/browse/term/HP:5200241).
Problematic substance use should be the label of the current term https://hpo.jax.org/app/browse/term/HP:0033511

The term Addictive behavior HP:0030858 should be obsoleted.

**Comment by pnrobinson** [comment]
@bcoleman96 Addictive behavior HP:0030858  actually seems like a reasonable term. What was the objection?

**Comment by pnrobinson** [comment]
@bcoleman96 Let's not obsolete this term. I think we will be reviewing this terms at a future workshop, which will be the better place to discuss potential changes like this. Please reopen if needed.

**Comment by bcoleman96** [comment]
I think the major issue is that we can't easily differentiate between addictive behavior and several other terms in this section, but I think this will need to be resolved in a workshop.
